J/JlrsLedger/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "JlrsLedger"
+version = v"0.0.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/Taaitaaiger/jlrs_ledger.git",
+              "5ff67d8b0f6b6eea50904290e9888e8779e7cd8f"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/jlrs_ledger
+
+if [[ "${target}" == *-musl ]]; then 
+    rust_flags="-C target-feature=-crt-static"
+fi
+
+RUSTFLAGS=$rust_flags cargo build --release --verbose
+
+install_license LICENSE
+if [[ "${target}" == *-mingw* ]]; then
+    install -Dvm 0755 "target/${rust_target}/release/jlrs_ledger.${dlext}" "${libdir}/libjlrs_ledger.${dlext}"
+else
+    install -Dvm 0755 "target/${rust_target}/release/libjlrs_ledger.${dlext}" "${libdir}/libjlrs_ledger.${dlext}"
+fi
+"""
+
+platforms = supported_platforms()
+
+is_supported(p) = arch(p) ∈ ("i686", "x86_64") &&
+                  (Sys.islinux(p) || nbits(p) == 64 && Sys.iswindows(p))
+
+filter!(is_supported, platforms)
```

Comment:
Can slightly simplify to
```suggestion
is_supported(p) = arch(p) ∈ ("i686", "x86_64") &&
                  (Sys.islinux(p) || nbits(p) == 64 && Sys.iswindows(p))
platforms = supported_platforms(; exclude=!is_supported)
```
I'm a bit curious about only these few platforms

Reply:
Thanks for the advice! 

The main reason is that this library is only intended to be used in combination with jlrs, and these are the only platforms jlrs currently supports.